TrainingTask/Task2.EmployeeCollection/CollectionManagement.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TrainingTask.Task2.EmployeeCollection
+{
+	public class CollectionManagement
+	{
+		public void Execute()
+		{
+			var employees = Employee.GetEmployees();
+			//
+			// Last name sorting.
+			//
+
```

Comment:
Please put comments exactly before the logic. Do not add these empty lines after comment block.